php/class-plugin-abstract.php

```diff
@@ -53,6 +53,14 @@ abstract class Plugin_Abstract implements Plugin_Interface {
 	 */
 	protected $slug;
 
+	/**
+	 * The slug of the post type that stores the blocks.
+	 *
+	 * @since 1.3.5
+	 * @var string
+	 */
+	protected $post_type_slug = 'block_lab';
```

Comment:
Do you think this belongs in the plugin abstract? I imagine it would go in the `Plugin` class, not the abstract version of it (which to me, should be portable to different plugins).

Reply:
Ah, good point. Abstract should apply to any plugin.

Reply:
Good idea, https://github.com/getblocklab/block-lab/pull/384/commits/4614c78849587a6a2dc0b4ab04e287691212979b moves this to `Plugin`.